book/website/collaborating_github/reviewing-contributions.md

```diff
@@ -0,0 +1,89 @@
+
+## Guidelines for Review Process and Maintainance
+> For project collaboration on GitHub, it is very important to follow certain guidelines and best practices while maintaining a particular project. These guidelines become extremely important when the collaboration is done remotely on any project where the contributors and the maintainers might experience communication gaps due to various reasons. In such situations, contributing and maintenance can be done smoothly by the help of the following:
+
+1. Documenting the process- Writing down a guide on how contributors can get started with the project and how they can make meaningful contributions is the first step to be taken while maintaining any open source project.  
+A project's [README](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-readmes) file should throw light on what the project is all about, why was it created in the first place, who maintains the project and who can provide guidance to new contributors.
+
+2. Effective communication- The conversations regarding any contribution can be made public for others to join for a discussion while working on small features and ideas. This will involve more people and ensure transparency in the open source work.  
+It is important to write messages and comments, in a clear and easy to understand manner while doing a review to help the contributors in understanding the requirements so that they can make better commits to their pull requests.  
+It is better to mention important links in the messages if required.
+
+3. Mentorship and descriptions- The maintainer's job is to make contribution a process as easy as possible. Open source contributions can be intimidating for a lot of people.  
+Guiding people by making friendly and encouraging conversations can make the onboarding process for new contributors comfortable. It is better to create descriptive issues to resolve. For significant contributions to be made, it is better to create different issues before resolving them with PRs.  
+Labelling issues and PRs will help in getting more contributors involved in various tasks with different skill requirements. Labelling seemingly easy issues as ["good first issue"](https://help.github.com/en/github/building-a-strong-community/encouraging-helpful-contributions-to-your-project-with-labels) will help new contributors to pick up easy tasks like minor changes in code and content,bug and typo fixes and small design issues.
+
+4. Maintaining the PRs- The maintenance of already existing PRs on a project involves labelling them, reviewing them, changing their stages, merging and closing them. Maintenance of PRs can be done by giving the right review on the right time.  
+It is ideal for any maintainer to review a contribution in the form of a PR within one week for an active project and provide labels as soon as possible.
```

Comment:
```suggestion
It is ideal for any maintainer to review a contribution in the form of a PR within one week for an active project and provide labels as soon as possible.
```
This could set the expectation that all projects try to merge within one week whereas that is dependent on the size and workload of the maintenance team. Can we say something like "Maintainers may communicate a timescale within which they try to merge a PR, for example one week" instead?

Reply:
> Can we say something like "Maintainers may communicate a timescale within which they try to merge a PR, for example one week" instead?  

This is a very apt way to make changes in this point. I have created a [commit](https://github.com/alan-turing-institute/the-turing-way/pull/1189/commits/e92a3593c1b095b5eec740994bd5f520c2e3b79f) related to this. Please check once.

Reply:
Looks great, thanks!